.eslintrc

```diff
@@ -70,6 +70,36 @@
         "import/first": 0,
       },
     },
+    {
+      "files": [
+        "src/Components/Intercom.js",
+      ],
+      "rules": {
+        "class-methods-use-this": 0,
```

Comment:
Is this rule needed for `Intercom.js`?

Reply:
I thought the eslint error was thrown by the use of `window.`, in the intercom.js, but thats not the case. It's because ESLint suggest a static method instead of an object method for `Intercom#getIntercomAppId` (https://github.com/Scrivito/scrivito_example_app_js/pull/47/files#diff-96ce2237efc203044372d3051417e295R35). So i will change it to a static one, so we can get rid of the `.eslintrc` entry 👍